pkg/clusterapi/config/containerd_config_append.toml

```diff
@@ -0,0 +1,7 @@
+[plugins."io.containerd.grpc.v1.cri".registry.mirrors]
+    [plugins."io.containerd.grpc.v1.cri".registry.mirrors."public.ecr.aws"]
+    endpoint = ["https://{{.registryMirrorAddress}}"]
+    {{- if .registryCACert }}
```

Comment:
Why is this red?

Reply:
not sure, guess `toml` file type does not detect template format

Reply:
indentation issue, updated

Reply:
nvm idk maybe just file type highlight